packages/kbn-openapi-bundler/src/openapi_bundler.test.ts

```diff
@@ -6,125 +6,235 @@
     });
 
     it('makes properties in a referenced object node partial', async () => {
-      await bundleFolder('modify_partial_ref');
-      await expectBundleToMatchFile('modify_partial_ref', 'expected.yaml');
+      const folder = join('x_modify', 'modify_partial_ref');
+
+      await bundleFolder(folder);
+      await expectBundleToMatchFile(DEFAULT_BUNDLED_FILE_PATH, join(folder, 'expected.yaml'));
     });
 
     it('makes properties in an object node required', async () => {
-      await bundleFolder('modify_required_node');
-      await expectBundleToMatchFile('modify_required_node', 'expected.yaml');
+      const folder = join('x_modify', 'modify_required_node');
+
+      await bundleFolder(folder);
+      await expectBundleToMatchFile(DEFAULT_BUNDLED_FILE_PATH, join(folder, 'expected.yaml'));
     });
 
     it('makes properties in a referenced object node required', async () => {
-      await bundleFolder('modify_required_ref');
-      await expectBundleToMatchFile('modify_required_ref', 'expected.yaml');
+      const folder = join('x_modify', 'modify_required_ref');
+
+      await bundleFolder(folder);
+      await expectBundleToMatchFile(DEFAULT_BUNDLED_FILE_PATH, join(folder, 'expected.yaml'));
     });
   });
 
-  describe('x-inline', () => {
-    it('inlines a reference', async () => {
-      await bundleFolder('inline_ref');
-      await expectBundleToMatchFile('inline_ref', 'expected.yaml');
+  describe('inline references', () => {
+    it('inlines local references', async () => {
+      const folder = join('inline_ref', 'inline_local_ref');
+
+      await bundleFolder(folder);
+      await expectBundleToMatchFile(DEFAULT_BUNDLED_FILE_PATH, join(folder, 'expected.yaml'));
+    });
+
+    it('inlines external references', async () => {
+      const folder = join('inline_ref', 'inline_external_ref');
+
+      await bundleFolder(folder);
+      await expectBundleToMatchFile(DEFAULT_BUNDLED_FILE_PATH, join(folder, 'expected.yaml'));
     });
   });
 
   describe('skip internal', () => {
     it('skips nodes with x-internal property', async () => {
-      await bundleFolder('skip_internal');
-      await expectBundleToMatchFile('skip_internal', 'expected.yaml');
+      const folder = join('skip', 'skip_internal');
+
+      await bundleFolder(folder);
+      await expectBundleToMatchFile(DEFAULT_BUNDLED_FILE_PATH, join(folder, 'expected.yaml'));
     });
 
     it('skips endpoints starting with /internal', async () => {
-      await bundleFolder('skip_internal_endpoint');
-      await expectBundleToMatchFile('skip_internal_endpoint', 'expected.yaml');
+      const folder = join('skip', 'skip_internal_endpoint');
+
+      await bundleFolder(folder);
+      await expectBundleToMatchFile(DEFAULT_BUNDLED_FILE_PATH, join(folder, 'expected.yaml'));
+    });
+  });
+
+  describe('remove props', () => {
+    it('removes "x-codegen-enabled" property', async () => {
```

Comment:
Should we also check that other `x-` properties are getting removed?

Reply:
I've added extra tests.